installer/datafiles/ruby.data

```diff
@@ -3,38 +3,14 @@ RUBY_DEST:                '/opt/microsoft/omsagent/ruby'
 
 %Files
```

Comment:
Yes, Jeff, I did not include ruby installer changes here on purpose. These are not related to MySQL plugin feature work. As noted in the comments section, I  will be sending a separate code review for the ruby installer script and its documentation. Thank you for your review and feedback . Much appreciated.